mail_tracking_mass_mailing/hooks.py

```diff
@@ -3,26 +3,14 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 import logging
-from openerp import api, SUPERUSER_ID
+from openerp.addons.mail_tracking.hooks import column_add_with_value
```

Comment:
Yes, it is. This causes server doesn't start if you don't have both modules accessible.